src/modules/acbs/dto/acbs-get-facility-fixed-fee-response.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { DateString } from '@ukef/helpers';
+
+export type AcbsGetFacilityFixedFeeResponseDto = AcbsGetFacilityFixedFeeDto[];
+
+export interface AcbsGetFacilityFixedFeeDto {
```

Comment:
Can we go with `AcbsGetFacilityFixedFeeResponseItem` instead of `AcbsGetFacilityFixedFeeDto` please?